modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,19 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form. Note that you can import the SAML 2.0 metadata only when you are creating a new SAML 2.0 configuration.
```

Comment:
Is it possible to make this a two-step task, with numbered steps (maybe one to nav and one to click the link?)
Also, consider setting the restriction right from the beginning:
```suggestion
When creating a new SAML 2.0 configuration, you can import your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form. 
```

Reply:
Import the provider's metadata FILE? XML file?

Reply:
```suggestion
When creating a new SAML 2.0 configuration, you can import an XML file containing your identity provider's SAML 2.0 metadata to Anypoint Platform using the *Import IdP Metadata* link. Once uploaded, the XML file's values automatically populate the *Identity Management SAML 2.0* form.
```